lib/bolt_server/schemas/task.json

```diff
@@ -59,6 +84,6 @@
       "additionalProperties": false
     }
   },
-  "required": ["name", "file"],
+  "required": ["name"],
```

Comment:
Is there a way to express that either 'file' or 'files' is required?

Reply:
It's possible but kind of verbose and "file" should be rejected once everything else is promoted. I'll add a TODO

Reply:
oh yeah it's JSON I can't, thats why I didn't add one.